validation/standard/standard-bytecodes.toml

```diff
@@ -28,7 +48,7 @@ preimage_oracle = "0x8b26e228dfe0bb4d7dee801d50756792a7f2d63ea5da42ddb120873dfe4
 permissioned_dispute_game = "0xb0a3aed0fb964b33445061d604e5fcfb952fb99dbff4d6f0c59aa1f24fdfd30f"       # version "1.2.0"
 # for the contracts below which contain immutables, the hash is calculated from the masked bytecode (not the deployed bytecode as-is).
 # sections of the bytecode that contain the immutables are masked (set to 0) before computing the hash
-anchor_state_registry = "0x07a466697584157d8926768fd99d30499dddc52aec7e2e620f430f3498b26dee" # version "1.0.0"
+anchor_state_registry = "0x46e8ecc46fd27f178ada1c5a2168c09512e7ed2f7d4ac9a7c74123af26fd850c" # version "2.0.0"
```

Comment:
Given this was incorrect before, wondering why no checks on chains were failing. Is it because none were on `op-contracts/v1.6.0`?

Reply:
We don't check bytecodes for testnets, because we don't require their contract versions to be exactly equal to the standard release contract versions.

Reply:
OP Sepolia is now on `v1.8.0` though, and I've manually confirmed it would pass all the standard validations with the changes on this branch. This puts us in a good spot for mainnet activation, since the bytecodes will be the same.